changelogs/fragments/57464-set-ovirt-disk-active-default.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - Fix when creating disk for vm its not activated (https://github.com/ansible/ansible/issues/57412)
```

Comment:
Please make it more understandable, for example:
"Fix the issue that disk is not activated after its creation"